plugins/content-length.rst

```diff
@@ -1,4 +1,38 @@
 Content-Length Plugin
 =====================
 
-TODO
+The Content-Length Plugin allows you to make your HTTP Request compliant with most of the servers.
+It's main purpose is to set the correct `Content-Length` Header value based on your stream size.
+
+However if it's failed to get the size of a stream, it will set the correct header and decorate the body of the request
+with Chunked transfer encoding as defined in the `RFC 2145`_::
```

Comment:
If the size of the stream can not be determined, the plugin sets the `Encoding` header to `chunked`, as defined in RFC 2145. 

for the rfc's, we used a sphinx alias in FOSHttpCacheBundle, see for example https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/blob/master/Resources/doc/reference/configuration/headers.rst (and look for :rfc:)

@ddeboer what do we need to set up to make `:rfc:` work? i can't find the configuration anymore.